BrainPortal/test_api/req_files/groups/badget/out

```diff
@@ -0,0 +1,2 @@
+404 application/json  \[.*
```

Comment:
Why is there a regex for a elimination pattern that will never happen in the output?

Reply:
This one no idea, actually I think you suggested it. Should I delete?

Reply:
Yes please, no need to confuse future testers about why that's there, especially since there are no square brackets in the output anymore.

Yes I did suggest it, but that was appropriate at the time, it allowed comparisons of the outputs before you changed the outputs to something more deterministic.